test/integration/targets/openvswitch_bridge/tests/basic.yaml

```diff
@@ -22,5 +22,27 @@
     that:
       - "result.changed == false"
 
-- name: Tear down test bridge
+- name: Add fake bridge
+  openvswitch_bridge:
+    bridge: fake-br-test
+    parent: br-test
+    vlan: 100
+  register: result
+
+- assert:
+    that:
+      - "result.changed == true"
+
+- name: Change fake bridge vlan
+  openvswitch_bridge:
+    bridge: fake-br-test
+    parent: br-test
+    vlan: 300
+  register: result
+
+- assert:
+    that:
+      - "result.changed == true"
```

Comment:
```suggestion
      - result.changed
```

Reply:
All right, let's change it, although it is unrelated to the patch scope!